core/services/commonwealth/README.md

```diff
@@ -0,0 +1,3 @@
+# Commonwealth of Services
```

Comment:
What is the reason to have an empty .gitignore ?
We can add it when necessary.

Reply:
No specific reason besides o.c.d :smile: